examples/multigpu_advtrain/make_model.py

```diff
@@ -0,0 +1,35 @@
+from model import MLP
+from model import Conv2D
+from model import ReLU
+from model import Flatten
+from model import Linear
+from model import Softmax
+from model import MaxPool
+
+from resnet_tf import ResNetTF
+
+
+def make_madry(nb_classes=10, input_shape=(None, 28, 28, 1), **kwargs):
```

Comment:
we already have the Madry Lab Challenge model in another example. Should we just import that one, or is this one different somehow?

Reply:
This one uses the multigpu layers. Clarified in 8e181e5 and 0861a8b.